jdaviz/configs/imviz/helper.py

```diff
@@ -398,8 +398,12 @@ def layer_is_2d(layer):
     return isinstance(layer, BaseData) and layer.ndim == 2
 
 
+def layer_is_2d_or_3d(layer):
+    return isinstance(layer, BaseData) and layer.ndim in (2, 3)
+
+
 def layer_is_image_data(layer):
-    return layer_is_2d(layer) and not layer.meta.get(_wcs_only_label, False)
+    return layer_is_2d_or_3d(layer) and not layer.meta.get(_wcs_only_label, False)
```

Comment:
Doing a search of the codebase, I don't expect this to have any actual consequences anywhere.  For Imviz, there is no ability to load cube data, and for cubeviz whenever we call this we really want to include cubes.

Reply:
But what about in the future when we want to support "flexible viz" and could possibly have 2D and 3D data both loaded and Imviz also loaded?

Reply:
we can rename this method if that would help (suggestions welcomed!)... but for now, all uses of it actually mean something that appears like an image on the screen (so either 2d or 3d image data).